assets/js/components/sell-quick-start.component.js

```diff
@@ -44,19 +44,26 @@ function sellQuickStartController ($scope, $rootScope, currency, buySell, Alerts
     $scope.sellLimit = $scope.exchange._profile._currentLimits._bank._outRemaining.toString();
   }
 
-  const setInitialCurrencyAndSymbol = (code, name) => {
-    $scope.sellTransaction.currency = { code: code, name: name };
-    $scope.sellCurrencySymbol = currency.conversions[code];
-    $scope.limitsCurrencySymbol = currency.conversions[code];
+  $scope.initializeCurrencyAndSymbol = () => {
+    const setInitialCurrencyAndSymbol = (code, name) => {
+      $scope.sellTransaction.currency = { code: code, name: name };
+      $scope.sellCurrencySymbol = currency.conversions[code];
+      $scope.limitsCurrencySymbol = currency.conversions[code];
+    };
+
+    if ($scope.exchangeCountry === 'DK') {
+      setInitialCurrencyAndSymbol('DKK', 'Danish Krone');
+    } else if ($scope.exchangeCountry === 'GB') {
+      setInitialCurrencyAndSymbol('GBP', 'Great British Pound');
+    } else {
+      setInitialCurrencyAndSymbol('EUR', 'Euro');
+    }
+    $scope.$watch('sellTransaction.currency', (newVal, oldVal) => {
```

Comment:
This will add a new watcher whenever `$scope. initializeCurrencyAndSymbol` is called, intentional?